index.html

```diff
@@ -100,6 +100,41 @@ <h2>A project for first-time contributions</h2>
             </ul>
           </div>
         </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->        
+        
+        <!-- ________ Brooklyn Baylis Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Brooklyn Baylis</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/DevBrook" target="_blank">@DevBrook</a>
+          </p>
+          <p class="about">
+            I'm a 14 year old programmer, I am currently learning full stack web dev (HTML, CSS, JS) + Data Structures and Algorithms.
+            I know Python & C# and I use Unity to create games. 
+            I enjoy football, rugby, coding, reading, cars, anime/manga, animals, nature and much more!
+          </p>
+          <div class="resources">
+            <p>3 Useful Web Developement Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.udemy.com/course/design-and-develop-a-killer-website-with-html5-and-css3/" target="_blank" title="HTML & CSS Udemy Course">
+                  Udemy Course
+                </a>
+              </li>
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="Free web dev resource">
+                  FreeCodeCamp
+                </a>
+              </li>
+              <li>
+                <a href="https://developer.mozilla.org/en-US/" target="_blank" title="Free web dev documentation">
+                  MDN Web Docs
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
         <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
@DevBrook You have added your card at wrong place. You are supposed to add your card after the comment `<!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->`, this comment is below your card. So move your card below this comment and push your changes. We will merge it.